### PR TITLE
Fix permission check in utls.py (get_backreferences). It's "View" not "zope2.View"

### PR DESCRIPTION
**Pull request by maethu**
Before only the "Manager" role was able to see referenced members.

### CODE REVIEW

File: ftw/contacts/tests/test_utils_get_backreferences.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def test_do_not_append_objs_with_no_permission(self):
         self.assertEqual(
             [member_block], get_backreferences(contact, IMemberBlock))
 
+        member_block.manage_permission('View', roles=[])
```
Comment (elioschmutz) [inline ftw/contacts/tests/test_utils_get_backreferences.py]: Why is this necessary?

Comment (maethu) [inline ftw/contacts/tests/test_utils_get_backreferences.py]: This does actually removes all roles with view permissions and blocks the acquisition. 
Without this change, test is failing. 

Check rolemanager.py
```
    def manage_permission(self, permission_to_manage, roles=[], acquire=0):
        """Change the settings for the given permission.

        If optional arg acquire is true, then the roles for the permission
        are acquired, in addition to the ones specified, otherwise the
        permissions are restricted to only the designated roles.
        """

```
